copier.yml

```diff
@@ -40,7 +44,8 @@ distribution_name:
     type: str
     help: |
         Name of the python distribution package that will be created.
-        This is what people will `pip install`
+        This is what people will `pip install`.
+        It is generally the same as the repo name unless there is a name clash on PyPI.
```

Comment:
VERY predantic but it's odd to have the "repo_name" help start with "Generally" and this one start with "It is generally" as it mixes styles.

Reply:
```suggestion
        Generally the same as the repo name unless there is a name clash on PyPI.
```
How about this?

Reply:
Good with me